edge-util/src/Microsoft.Azure.Devices.Edge.Util/Proxy.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.Azure.Devices.Edge.Util
+{
+    using System;
+    using System.Net;
+    using Microsoft.Extensions.Logging;
+
+    public static class Proxy
+    {
+        public static Option<IWebProxy> Parse(string proxyUri, ILogger logger)
+        {
+            if (string.IsNullOrWhiteSpace(proxyUri))
```

Comment:
Nit: Either add check to logger being null or use it as logger?.LogInformation.. to avoid null refs

Reply:
Will do.

Reply:
Added check.

Reply:
...and test.